kloppy/tests/issues/issue_60/test_issue_60.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from kloppy.domain import Orientation
+from kloppy import opta
+
+
+def kloppy_load_data(f7, f24):
+    """
+
+    Args:
+        f7: filepath to the match details
+        f24: filepath to the event details with deleted event
+
+
+    Returns:
+        events: DataFrame consisting of all events
+        home_team_id: id of the home team
+        away_team_id: id of the away team
+
+    """
+    dataset = opta.load(f7_data=f7, f24_data=f24)
+
+    events = dataset.transform(
+        to_orientation=Orientation.FIXED_HOME_AWAY
+    ).to_df(
+        additional_columns={
+            "event_name": lambda event: str(getattr(event, "event_name", "")),
+            "player_name": lambda event: str(getattr(event, "player", "")),
+            "ball_state": lambda event: str(getattr(event, "ball_state", "")),
+            "team_name": lambda event: str(getattr(event, "team", "")),
+        },
+    )
+
+    metadata = dataset.metadata
+    home_team, away_team = metadata.teams
+    return events, home_team.team_id, away_team.team_id
+
+
+class TestIssue60:
+    def test_parse_opta(self):
```

Comment:
Please make sure in this test that the dataset does not contain the deleted events, but does include the other ones.

Reply:
Hi Koenvo, I updated Optaf24 to contain deleted events. This was done to test if my changes were working. I am not quite sure why it needs to be removed. Sorry to be a bother.

Reply:
First: thanks for this PR and sorry for the delay. 

Second: I understand I wasn't very clear with my comment. Please make sure in your test that you test the dataset does not contain any events that should be filtered out because they are deleted.

You can find an example here: https://github.com/PySport/kloppy/blob/master/kloppy/tests/test_event.py#L55-L66